purescript/src/Item.purs

```diff
@@ -0,0 +1,14 @@
+module Item where
+
+data Item =
```

Comment:
This can (and should if you ask me) be a `newtype` instead of `data`.

Reply:
Thank you! This is exactly the kind of feedback I need!

Would that be the same for all of the `type` in the code?

Reply:
Hmm, now when looking at `Item` i think it should keep being a `data` since the idea is that their could be more then one 
´Item´ type